### PR TITLE
Added AWS KMS support info

### PR DESCRIPTION
**Pull request by ZhmakaAS**
Added basic information about the new Acra-supported flags and KMS-related configuration file format. 

I think this PR should be marked as a draft until we extend all our Acra tools with supporting KMS for Acra Master Key loading as well.

### CODE REVIEW

File: content/acra/configuring-maintaining/general-configuration/acra-server/_index.md [hunk 1]
```diff
@@ -419,6 +419,24 @@ For additional certificate validation flags, see corresponding pages:
 [OCSP](/acra/configuring-maintaining/tls/ocsp/) and
 [CRL](/acra/configuring-maintaining/tls/crl/).
 
+### KMS
+
+* `--kms_type=<type>`
+
+  KMS type for using.
```
Comment (vixentael) [inline content/acra/configuring-maintaining/general-configuration/acra-server/_index.md:426]:
```suggestion
  Specify your KMS.
```

File: content/acra/configuring-maintaining/general-configuration/acra-server/_index.md [hunk 2]
```diff
@@ -419,6 +419,24 @@ For additional certificate validation flags, see corresponding pages:
...
+  Currently, supported only:
```
Comment (vixentael) [inline content/acra/configuring-maintaining/general-configuration/acra-server/_index.md:427]:
```suggestion
  Currently supported KMS types:
```

File: content/acra/configuring-maintaining/general-configuration/acra-server/_index.md [hunk 3]
```diff
@@ -419,6 +419,24 @@ For additional certificate validation flags, see corresponding pages:
...
+  * `aws` - AWS Key Management Service 
+
+* `--kms_credentials_path=<filepath>`
+
+  KMS credentials file path in JSON format.
```
Comment (vixentael) [inline content/acra/configuring-maintaining/general-configuration/acra-server/_index.md:432]:
```suggestion
  A path to a file with KMS credentials JSON format.
```

**Review by vixentael — CHANGES_REQUESTED** [review]

**Review by vixentael — COMMENTED** [review]
👏

**Review by vixentael — APPROVED** [review]

**Comment by ZhmakaAS** [comment]
Still need to add a similar section to the `log-verifier` tools when we add corresponding functionality.

**Comment by vixentael** [comment]
what are we waiting here? @ZhmakaAS @Lagovas

**Comment by Lagovas** [comment]
> what are we waiting here? @ZhmakaAS @Lagovas

merging https://github.com/cossacklabs/acra/pull/554. But #554 ready for merging too) If I haven't forgotten anything.

**Comment by vixentael** [comment]
lets merge then 😏

**Pull request merged**